newdle/migrations/versions/20190913_1031_000000000000_create_initial_tables.py

```diff
@@ -1,7 +1,7 @@
 """Create initial tables
 
 Revision ID: 000000000000
-Create Date: 2019-09-12 16:30:45.370144
+Create Date: 2019-09-13 10:31:54.565733
```

Comment:
nothing changed in the structure so it shouldn't be necessary to regenerate the revision